src/pas/plugins/oidc/plugins.py

```diff
@@ -112,6 +113,12 @@ class OIDCPlugin(BasePlugin):
             mode="w",
             label="Use a modified OpenID Schema for email_verified and phone_number_verified boolean values coming as string. ",
         ),
+        dict(
```

Comment:
IMHO `default_user_info` is not the best name/description for this purpose, my suggestion is to change it to something clearer like "User info property used as userid". And probably whoever configures the plugin should be warned that the user should not be able to freely change this property (otherwise he/she could easily impersonate another user).